content/en/docs/concepts/services-networking/service.md

```diff
@@ -657,7 +657,6 @@ Once set, it cannot be changed.
 The value of `spec.loadBalancerClass` must be a label-style identifier,
 with an optional prefix such as "`internal-vip`" or "`example.com/internal-vip`".
 Unprefixed names are reserved for end-users.
-You must enable the `ServiceLoadBalancerClass` feature gate to use this field.
```

Comment:
We should also update https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ - has that happened in a different pull request?

Reply:
Yes, that already merged in this PR: #28644